l10n_ch_hr_payroll/__init__.py

```diff
@@ -1,5 +1,5 @@
-# -*- coding: utf-8 -*-
-# © 2012 David Coninckx (Opennet Sarl)
-# License into __openerp__.py.
-
-from . import models
+# -*- coding: utf-8 -*-
+# © 2017 Open Net Sàrl
```

Comment:
still in all files © should be replaced.
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L2 & OCA/maintainer-tools#197